ansible/manage-node.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Manage epoch nodes
+  hosts: all
+  remote_user: epoch
+  vars:
+    project_root: "{{ ansible_env.HOME }}/node"
+    epoch_action: "{{ c | default('none') }}"
+  tasks:
+  - name: Checking if the input in correct
+    debug:
+      msg: "Incorrect input! You must input either 'start', 'stop' or 'restart'"
+    when: epoch_action != "start" and epoch_action != "stop" and epoch_action != "restart"
+
+  - name: Check if epoch binary exists
+    stat:
+      path: "{{ project_root }}/bin/epoch"
+    register: epoch
+
+  - name: Checking the epoch daemon state
+    command: "{{ project_root }}/bin/epoch ping"
+    failed_when: no
+    register: ping
+    when: epoch.stat.exists == True
+
+  - name: Starting the epoch deamon
+    command: "{{ project_root }}/bin/epoch start"
+    when: epoch.stat.exists == True and not ping.stdout == "pong" and epoch_action == "start"
+
+  - name: Stopping the epoch deamon
+    command: "{{ project_root }}/bin/epoch stop"
+    when: epoch.stat.exists == True and ping.stdout == "pong" and epoch_action == "stop"
+
+  - name: Restarting the epoch deamon
+    command: "{{ project_root }}/bin/epoch restart"
+    when: epoch.stat.exists == True and ping.stdout == "pong" and epoch_action == "restart"
```

Comment:
Restart command implies starting the node even if it's stopped, which this task unfortunately won't do.